diff --git a/dist/solid-polygon-layer/polygon-tesselator.d.ts.map b/dist/solid-polygon-layer/polygon-tesselator.d.ts.map
deleted file mode 100644
index a619818aa2126e2a24325e2945b7b1828130a6c3..0000000000000000000000000000000000000000
diff --git a/dist/solid-polygon-layer/polygon.d.ts.map b/dist/solid-polygon-layer/polygon.d.ts.map
deleted file mode 100644
index b299f36f00094039b1c34818b1a83813020b5101..0000000000000000000000000000000000000000
diff --git a/dist/solid-polygon-layer/solid-polygon-layer.d.ts.map b/dist/solid-polygon-layer/solid-polygon-layer.d.ts.map
deleted file mode 100644
index 9ed625b1c7b3145de0bf9ed7b03b9029aa9ab7ab..0000000000000000000000000000000000000000
diff --git a/src/solid-polygon-layer/polygon-tesselator.ts b/src/solid-polygon-layer/polygon-tesselator.ts
index 750213f0f612d883e88f828fab035768b3751bad..52e887bc7479a334f381152a121c98a9333c24a9 100644
--- a/src/solid-polygon-layer/polygon-tesselator.ts
+++ b/src/solid-polygon-layer/polygon-tesselator.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 // deck.gl
 // SPDX-License-Identifier: MIT
 // Copyright (c) vis.gl contributors
diff --git a/src/solid-polygon-layer/polygon.ts b/src/solid-polygon-layer/polygon.ts
index 630a5a940addae17499633cba0b1f0cf65230ef6..fbcc97a42dcea53dd1a5884b1cad25a1c3ebe306 100644
--- a/src/solid-polygon-layer/polygon.ts
+++ b/src/solid-polygon-layer/polygon.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 // deck.gl
 // SPDX-License-Identifier: MIT
 // Copyright (c) vis.gl contributors
