diff --git a/node_modules/carbon-components-svelte/src/Tabs/Tabs.svelte b/node_modules/carbon-components-svelte/src/Tabs/Tabs.svelte
index 51f088a..df41f53 100644
--- a/node_modules/carbon-components-svelte/src/Tabs/Tabs.svelte
+++ b/node_modules/carbon-components-svelte/src/Tabs/Tabs.svelte
@@ -204,7 +204,9 @@
       return reorderedContent;
     });
 
-    selected = currentIndex;
+    if (selected !== currentIndex) {
+      selected = currentIndex;
+    }
 
     if (prevIndex > -1 && prevIndex !== currentIndex) {
       dispatch("change", currentIndex);
