diff --git a/src/Select/SelectItem.svelte b/src/Select/SelectItem.svelte
index abedf3f130586469ab463903bf16ad84176c31c0..17872eab321e00fe29f5a94eade6e0ebb8b26c7c 100644
--- a/src/Select/SelectItem.svelte
+++ b/src/Select/SelectItem.svelte
@@ -42,21 +42,17 @@
   const id = uniqueId();
   const ctx =
     getContext("carbon:Select") || getContext("carbon:TimePickerSelect");
+  const selectedValue = ctx.selectedValue;
 
   $: {
     ctx?.setDefaultValue?.(id, value);
     ctx?.syncNativeSelectValue?.();
   }
 
-  let selected = false;
-
-  const unsubscribe = ctx.selectedValue.subscribe((currentValue) => {
-    selected = currentValue === value;
-  });
+  $: selected = $selectedValue === value;
 
   onMount(() => {
     return () => {
-      unsubscribe();
       ctx?.syncNativeSelectValue?.();
     };
   });
diff --git a/src/ComboBox/ComboBox.svelte b/src/ComboBox/ComboBox.svelte
--- a/src/ComboBox/ComboBox.svelte
+++ b/src/ComboBox/ComboBox.svelte
@@ -733,7 +733,6 @@
   <ListBox
     class={comboBoxListBoxClass}
     id={comboId}
-    aria-label={ariaLabel}
     aria-disabled={readonly || undefined}
     {disabled}
     invalid={showInvalid}
